components/server/src/ome/services/ThumbnailBean.java

```diff
@@ -839,9 +839,18 @@ public void createThumbnail(Integer sizeX, Integer sizeY)
      *
      * @return thumbnail object
      */
-    private Thumbnail _createThumbnail() {
-        // For old times sake
-        return _createThumbnail(thumbnailMetadata);
+    private Thumbnail _createThumbnail() throws ResourceError {
+        StopWatch s1 = new Slf4JStopWatch("omero._createThumbnail");
```

Comment:
We will now lose the StopWatch info  when ``_createThumbnail(thumbnailMetadata)`` is invoked in the new method.
so maybe in that case, keep ``StopWatch s1 = new Slf4JStopWatch("omero._createThumbnail");`` in `` _createThumbnail(thumbnailMetadata)``